lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -60,13 +60,6 @@ def cmd_run(*args, action: nil, opts: {})
     rhosts = mod_with_opts.datastore['RHOSTS']
     rhosts_walker = Msf::RhostsWalker.new(rhosts, mod_with_opts.datastore)
 
-    begin
```

Comment:
Before this PR - validation happens:

```
msf6 auxiliary(dos/http/cable_haunt_websocket_dos) > run 192.168.123.1 192.168.123.2 verbose=test
[-] Msf::OptionValidateError The following options failed to validate:
[-] Invalid option VERBOSE: for rhosts value 192.168.123.1, for rhosts value 192.168.123.2
```

After this PR - no validation happens:

```
msf6 auxiliary(dos/http/cable_haunt_websocket_dos) > run 192.168.123.1 192.168.123.2 verbose=test
[*] Running module against 192.168.123.1
[*] Running module against 192.168.123.2
[*] Auxiliary module execution completed
msf6 auxiliary(dos/http/cable_haunt_websocket_dos) > 
```

Reply:
Want to give it another shot?